app/api/search/documentQueryBuilder.js

```diff
@@ -86,18 +86,19 @@ export default function() {
       term,
       fieldsToSearch = ['title', 'fullText'],
       number_of_fragments = 1,
-      type = 'fvh',
-      fragment_size = 300
+      searchTextType = 'query_string'
     ) {
-      if (!term) {
+      if (!term || term === '*') {
```

Comment:
what happens if you put '**' as search term?

Reply:
@konzz, Good catch, I didn't test it, ** is returning the same "cannot convert undefined or null to object", in fact, whatever number of * throws the same error, do you think it is worth to extend the control for these?

Reply:
After debugging to identify the main reason for the error, It turns out that it was a problem with the processing of the response, for some reason when the search is one or more *, returned hits include some without snippets, given we were always expecting them, it was producing an exception, it's fixed now.